### PR TITLE
Julie new change for ReadMe.txt

### PR DESCRIPTION
**Pull request by JulieKoz**

### CODE REVIEW

File: ReadMe.txt [hunk 1]
```diff
@@ -15,4 +15,8 @@ Steve's Change
 
 
 Julie's change down here.
-I always go last.  Don't change it!!!!
\ No newline at end of file
+I always go last.  Don't change it!!!!
+
```
Comment (evilearthwormjim) [inline ReadMe.txt]: remove whitwspace

File: ReadMe.txt [hunk 2]
```diff
@@ -15,4 +15,8 @@ Steve's Change
...
+
+
+Julie's new change!
```
Comment (evilearthwormjim) [inline ReadMe.txt]: nice work 👍

**Review by evilearthwormjim — CHANGES_REQUESTED** [review]
Please see my white space request

**Review by evilearthwormjim — APPROVED** [review]

File: ReadMe.txt [hunk 3]
```diff
@@ -15,4 +15,8 @@ Steve's Change
 
 
 Julie's change down here.
```
Comment (Truiant) [inline ReadMe.txt:17]: Isn't there a Julie's change up there?

**Review by Truiant — COMMENTED** [review]
Done

**Pull request merged**